tslearn/neighbors.py

```diff
@@ -177,19 +177,21 @@ class KNeighborsTimeSeries(KNeighborsTimeSeriesMixin, NearestNeighbors,
     ----------
     n_neighbors : int (default: 5)
         Number of nearest neighbors to be considered for the decision.
-    metric : {'dtw', 'softdtw', 'euclidean', 'sqeuclidean', 'cityblock',
-              'sax'} (default: 'dtw')
+
+    metric : {'dtw', 'softdtw', 'euclidean', 'sqeuclidean', 'cityblock', 'sax'} (default: 'dtw')
```

Comment:
Any reason for this? You want to keep the possible values of the parameter on the same line as the name of the parameter? It does violate PEP-8.

Reply:
But then the docs do not look OK. I do not know how to break a long first line in sphinx/numpydoc specification of attributes.

Reply:
Could you try adding a backslash before the newline? 

https://stackoverflow.com/questions/14414878/how-do-i-break-a-link-in-a-rst-docstring-to-satisfy-pep8

Reply:
Worked! Thx!